mne/source_estimate.py

```diff
@@ -1405,6 +1401,20 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
             View to use. See surfer.Brain().
         colorbar : bool
             If True, display colorbar on scene.
+        clim : str | dict
```

Comment:
I'd just say `'auto' | dict` since that is the only string accepted. Can you look around in the repo to check how it is documented when there is only one acceptable string?